### PR TITLE
Setup a jvm configuration for Maven

### PR DESCRIPTION
**Pull request by imonteroperez**
Setting up a jvm configuration as code to run the testing

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by imonteroperez** [comment]
Requesting reviews from @timja @raul-arabaolaza

**Review by raul-arabaolaza — APPROVED** [review]
Changes LGTM, but I have no idea of how much resources the ATH uses on ci.jenkins.io. I have neverrun it with an OSS war so it will be great if someone with that knowledge reviews the values themselves

**Review by timja — COMMENTED** [review]
failure needs checking, I guess okay though

**Comment by imonteroperez** [comment]
> failure needs checking, I guess okay though

Seems to be unrelated Selenium failures, but not sure. BTW job was aborted by @dduportal. I have no perms to relaunch it again :(

**Comment by jtnord** [comment]
as this is not worsre than master in terms of failing tests I am going to merge this.  Note that the surefire VM still seems to be unconfigured.

File: .mvn/jvm.config [hunk 1]
```diff
@@ -0,0 +1 @@
+-Xmx512m -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1
```
Comment (jtnord) [inline .mvn/jvm.config:1]: I am assuming that this was just copied from elsewhere and no investigation was done on the max heap?

**Pull request merged**